src/ResourceManager/Compute/Commands.Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs

```diff
@@ -38,7 +38,7 @@ public partial class NewAzureRmDiskConfigCommand : Microsoft.Azure.Commands.Reso
             Position = 0,
             ValueFromPipelineByPropertyName = true)]
         [Alias("AccountType")]
-        public StorageAccountTypes? SkuName { get; set; }
+        public string SkuName { get; set; }
```

Comment:
@hyonholee out of curiosity, why the switch from this enum to string across all cmdlets?

Based on the markdown files, it seems that the accepted values went from `StandardLRS` and `PremiumLRS` to `Standard_LRS` and `Premium_LRS`, respectively. This would be a breaking change for existing scripts, which is fine since this will go out as a preview, but it might be worth looking at accepting the old values as well and handling them as special cases until you can make the breaking change in May. Let me know what you think.

Reply:
@cormacpayne  Yes, there are breaking changes in the Swagger spec and client library.  That is why we want preview module.